### ISSUE TITLE
Select Not working on Laravel 5.4 validate

### ISSUE THREAD

**Issue by dennist12** [Dogfalo/materialize#4175]
Here is may blade template
![blade](https://cloud.githubusercontent.com/assets/13460041/22560335/2b7ff632-e9af-11e6-8f00-59d31902816a.png)

Return request 
![error ressult](https://cloud.githubusercontent.com/assets/13460041/22560336/2b8123ae-e9af-11e6-9952-f5b398f2c8c8.png)

I use broswer-default class in select.
![with browser-default](https://cloud.githubusercontent.com/assets/13460041/22560337/2b8da0de-e9af-11e6-80ad-e01267e054df.png)

**Comment by developeranirudhprabhu** [comment]
@dennist12 Can you elaborate your issue along with these screenshots?

**Comment by dennist12** [comment]
The screenshot are the result of Laravel Request, the without the class browser-default and the last one with the class browser.

**Comment by Dogfalo** [comment]
We can't provide help for specific frameworks that we don't officially support.

**Comment by dennist12** [comment]
Thank you.

**Comment by aminnairi** [comment]
Using an identifier on the form tag and the attribute `form` on the select does fix the issue on Laravel 5.4.

```
<tbody>
    <tr>
        <form action="/market" method="POST" id="marketStore">
            <input type="hidden" name="_token" value="{{ csrf_token() }}">
            <td class="input-field">
                <input type="text" name="reference"> 
            </td>
            <td class="input-field">
                <select name="third_id" form="marketStore">
                    <option value="1" selected>Amin</option>
                </select>
            </td>
        </form>
    </tr>
</tbody>
```
